packages/analytics-react-native/src/react-native-client.ts

```diff
@@ -170,7 +174,7 @@ export class AmplitudeReactNative extends AmplitudeCore {
   }
 
   getSessionId() {
-    return this.config?.sessionId;
+    return this.explicitSessionId ?? this.config?.sessionId;
```

Comment:
I think we should revert this so it matches [browser-client](https://github.com/amplitude/Amplitude-TypeScript/blob/0f5fccc83d3f7f0a80adc4a0807fbd7e71c72e4a/packages/analytics-react-native/src/react-native-client.ts#L173).

I would expect this to set the `sessionId` but also adhere to other session rules and expire with sessionTimeout.

Reply:
ok, it depends on comment below

Reply:
Reverted